src/components/button/button.stories.tsx

```diff
@@ -0,0 +1,109 @@
+import { Meta, StoryObj } from '@storybook/react';
+import Button from './button';
+import { Plus } from 'lucide-react';
+
+// Button component story configuration
+const meta: Meta = {
+	title: 'Atoms/Button',
+	component: Button,
+	parameters: {
+		layout: 'centered',
+	},
+	tags: [ 'autodocs' ],
+	argTypes: {
```

Comment:
Do we need this args, when we have the interface defined?

Reply:
No. It's not required. 
By default, the union type shows `union` in the type summary, and it also shows Radio button controls. However, we can change the control types if we want.